api/query/metadata_cache.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package query
+
+import (
+	"net/http"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// MetadataMapCached is the local cache of Metadata in searchcache.
+var MetadataMapCached map[string][]byte = nil
+
+type searchcacheMetadataFetcher struct {
+	client *http.Client
```

Comment:
Constructing a `http.Client` is really cheap. I think we can construct a `&http.Client{Timeout: time.Second * 5}` directly in `Fetch` every time, which also has the extra benefit of having a single timeout definition.